### ISSUE TITLE
Discussion on position management and trade signal

### ISSUE THREAD

**Issue by hall130** [ZENALC/algobot#71]
Actually, I think we should add the position replace all in or all out, I mentioned before that buying in wide positions can become dangerous when the market trend fluctuates sharply. In my opinion, it is necessary to increase position management.
In terms of trading signals, When we and the implementation of position management, each position has its own stop loss price,
but the stop profit price is same, This is my idea, welcome to discuss

**Comment by ZENALC** [comment]
It's a good feature to have. So, do you propose we add a percentage of how much of assets to go in?

**Comment by hall130** [comment]
> 这是一个很好的功能。那么，您是否建议我们增加要投入的资产的百分比？

Of course, at the same time, I also think that the long and short signals can be added to the strategy in order to adjust the direction of the strategy. What do you think of this idea?
